container/responsive_ratios.go

```diff
@@ -0,0 +1,27 @@
+package container
+
+// Ratio is a helper type to define a fraction of a container. It's a float32 alias.
+type Ratio = float32
+
+// Some helpers to define common ratios.
+
+// FullRatio is the full size of the container.
+func FullRatio() Ratio { return 1.0 }
```

Comment:
Why not define constants?

Reply:
To not store the value. Functions are compiled only if they are used. It's, it seems, a good practice. Maybe I'm wrong.

Reply:
Constants are the same https://stackoverflow.com/a/38982889 Most Go I see uses constants

Reply:
Yes.

But for example, Fyne returns colors from function calls, like the Padding or like some several values that are "constant". I propose functions to ensure that we can adapt the ratios from possible futur configuration. For example, following themes.

If it's really a problem, I can of course change the code and set it as constants.

Reply:
The reason behind theme returning values from a function is that "primary" colour could be respresented by any colour - changed at runtime. However isn't the concept of "TwoThird" always going to be 2/3? If the names were somehow semantic the discussion would track but their names do indicate staticness...